infra/dshop-control-client/src/pages/Edit.js

```diff
@@ -18,19 +18,32 @@ import Products from 'pages/Edit/Products'
 import ProductAdd from 'pages/Edit/ProductAdd'
 import ProductEdit from 'pages/Edit/ProductEdit'
 import Settings from 'pages/Edit/Settings'
-import Navigation from 'components/Edit/Navigation'
+import SignIn from 'pages/SignIn'
+//import Navigation from 'components/Edit/Navigation'
+import Navigation from 'components/Navigation'
```

Comment:
Doesn't look like the component was moved?

Reply:
Yikes, forgot to commit a few files.  Thanks for catching this.  Need to clean up my working tree.